modules/platform/rest-api/nuxeo-rest-api-io/src/main/java/org/nuxeo/ecm/restapi/jaxrs/io/capabilities/CapabilitiesJsonWriter.java

```diff
@@ -19,37 +19,33 @@
 
 package org.nuxeo.ecm.restapi.jaxrs.io.capabilities;
 
-import static org.nuxeo.common.function.ThrowableConsumer.asConsumer;
 import static org.nuxeo.ecm.core.io.registry.reflect.Instantiations.SINGLETON;
 import static org.nuxeo.ecm.core.io.registry.reflect.Priorities.REFERENCE;
 
 import java.io.IOException;
 
 import org.nuxeo.ecm.core.io.marshallers.json.ExtensibleEntityJsonWriter;
 import org.nuxeo.ecm.core.io.registry.reflect.Setup;
+import org.nuxeo.runtime.capabilities.Capabilities;
 
 import com.fasterxml.jackson.core.JsonGenerator;
 
 /**
  * @since 11.5
  */
 @Setup(mode = SINGLETON, priority = REFERENCE)
-public class ServerInfoJsonWriter extends ExtensibleEntityJsonWriter<ServerInfo> {
+public class CapabilitiesJsonWriter extends ExtensibleEntityJsonWriter<Capabilities> {
 
-    public static final String ENTITY_TYPE = "server";
+    public static final String ENTITY_TYPE = "capabilities";
 
-    public ServerInfoJsonWriter() {
-        super(ENTITY_TYPE, ServerInfo.class);
+    public CapabilitiesJsonWriter() {
+        super(ENTITY_TYPE, Capabilities.class);
     }
 
     @Override
-    protected void writeEntityBody(ServerInfo entity, JsonGenerator jg) throws IOException {
-        jg.writeStringField("distributionName", entity.getDistributionName());
-        jg.writeStringField("distributionVersion", entity.getDistributionVersion());
-        jg.writeStringField("distributionServer", entity.getDistributionServer());
-        entity.getHotfixVersion().ifPresent(asConsumer(h -> jg.writeStringField("hotfixVersion", h)));
-        jg.writeBooleanField("clusterEnabled", entity.isClusterEnabled());
-        jg.writeStringField("clusterNodeId", entity.getClusterNodeId());
-
+    protected void writeEntityBody(Capabilities entity, JsonGenerator jg) throws IOException {
+        for (var entry : entity.get().entrySet()) {
+            jg.writeObjectField(entry.getKey(), entry.getValue());
+        }
```

Comment:
```suggestion
        entity.get().entrySet().forEach(e -> jg.writeObjectField(e.getKey(), e.getValue());
```

Reply:
I added the `ThrowableBiConsumer` functional interface as `writeObjectField` throws an `IOException` in order to do the forEach directly on the Map.